.github/workflows/anaconda_pkg_build.yml

```
@@ -69,3 +69,28 @@ jobs:
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
           push: ${{ github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/tags') }}
+
+      - name: Docker meta for Concourse
+        id: concourse-meta
+        uses: docker/metadata-action@3a3bb3a81753dc99f090d24ee7e5343838b73a96
+        with:
+          images: |
+            continuumio/anaconda-pkg-build
+            public.ecr.aws/y0o4y9o3/anaconda-pkg-build
+          tags: |
+            type=ref,event=branch,suffix=-amd64
+            type=ref,event=pr,suffix=-amd64
+            type=match,pattern=pkg-build-(.*),group=1,suffix=-amd64
```

To avoid overwriting the latest tag, which points to the multiarch image:
```suggestion
          tags: |
            type=ref,event=branch,suffix=-amd64
            type=ref,event=pr,suffix=-amd64
            type=match,pattern=pkg-build-(.*),group=1,suffix=-amd64
          flavor: |
            latest=false
```